src/content/learn/scaling-up-with-reducer-and-context.md

```diff
@@ -669,11 +669,11 @@ ul, li { margin: 0; padding: 0; }
 
 </Sandpack>
 
-In the next step, you will remove prop passing.
+Pada langkah selanjutnya, Anda akan menghapus pengoperan *prop*.
 
-### Step 3: Use context anywhere in the tree {/*step-3-use-context-anywhere-in-the-tree*/}
+### Langkah 3: Gunakan *context* di mana saja dalam *tree* {/*step-3-use-context-anywhere-in-the-tree*/}
```

Comment:
```suggestion
### Langkah 3: Gunakan *context* di mana saja dalam pohon {/*step-3-use-context-anywhere-in-the-tree*/}
```

We can translate *tree* into pohon subsequently because we have introduced the word multiple times above.